packs/data/extinction-curse-bestiary.db/thessekka.json

```
@@ -1865,7 +2071,26 @@
                 "requirements": {
                     "value": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "key": "FlatModifier",
+                        "label": "Expanded Splash",
+                        "predicate": [
+                            "splash"
+                        ],
+                        "selector": "strike-damage",
+                        "value": "@actor.abilities.int.mod"
+                    },
+                    {
+                        "key": "Note",
+                        "predicate": [
+                            "splash"
+                        ],
+                        "selector": "strike-damage",
+                        "text": "The bomb deals splash damage to every creature within 10 feet of the target.",
+                        "title": "{item|name}"
+                    }
```

```suggestion
                    {
                        "key": "Note",
                        "predicate": [
                            "weapon:trait:splash"
                        ],
                        "selector": "strike-damage",
                        "text": "{item|system.description.value}",
                        "title": "{item|name}"
                    }
```
Splash damage needs a note for it, and we want to minimize localization.